editor/editor_config.cpp

```diff
@@ -129,15 +129,21 @@ bool EditorConfig::GetTypeDescription(vector<string> classificatorTypes,
       classificatorTypes.erase(it);
       break;
     }
+    // Adding partial types for 2..N-1 parts of a N-part type.
+    auto hyphenPos = it->find('-');
+    while ((hyphenPos = it->find('-', hyphenPos + 1)) != string::npos)
+    {
```

Comment:
No need here.

Reply:
I decided to add it since it's the last statement in the `for` loop, so it looks better.